main/src/cgeo/geocaching/utils/LifecycleStartedBroadcastReceiver.java

```diff
@@ -0,0 +1,38 @@
+package cgeo.geocaching.utils;
+
+import android.content.BroadcastReceiver;
+import android.content.Context;
+import android.content.IntentFilter;
+
+import androidx.annotation.NonNull;
+import androidx.lifecycle.DefaultLifecycleObserver;
+import androidx.lifecycle.LifecycleOwner;
+import androidx.localbroadcastmanager.content.LocalBroadcastManager;
+
+public abstract class LifecycleStartedBroadcastReceiver extends BroadcastReceiver implements DefaultLifecycleObserver {
```

Comment:
IMHO the name `LifecycleStartedBroadcastReceiver` is kinda misleading. What about `Life cycleAwareBroadcastReceiver`?